hackforla/VRMS
# Magic link 2

STEPS TO VERIFY: 
1. Pull my repo.
2. docker-compose build & docker-compose up
3. Log into the localhost:3000 page for the main app
4. Select the login button. Provide your email.
5. Log into your email and use the link provided. The link should direct you to the login page.
6. If you are not redirected to the login admin screen, then just refresh or login to the main page and press login again.

TODOs:
- Alex to determine if the approach is valid.
- Determine if we need to add a logout button now.
- Possibly make the user experience better with a redirect.
- Review the Tinyurl redirection for localhost
- Whatever else you think.
